lib/widgets/mapCardHero_widget.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:gra_terenowa/controller/mapData_controller.dart';
+import 'package:gra_terenowa/statics/colors.dart';
+import 'package:gra_terenowa/statics/constants.dart';
+import 'package:gra_terenowa/view/mapCard_screen.dart';
+
+class MapCardHero extends StatelessWidget {
+  const MapCardHero({Key? key, required this.onTap, required this.mapItemIndex})
+      : super(key: key);
+
+  final VoidCallback onTap;
+  final int mapItemIndex;
+
+  @override
+  Widget build(BuildContext context) {
+    final MapDataController _mapDataController = Get.find();
+
+    return Card(
+      elevation: 8,
+      color: AppColors.primaryWhite,
+      margin: EdgeInsets.all(Constants.cardMargin),
+      clipBehavior: Clip.antiAliasWithSaveLayer,
+      child: InkWell(
+        onTap: () {
+          onTap();
```

Comment:
It would be nice to recenter this scrollable view to the selected card, as it happens when selecting via the icons.

Reply:
I don't know if I understand you correctly, but my basic UI idea is to keep the map and bottom list seperate in interaction. That means that selecting something on the map will automatically scroll the bottom list view, but will not do anything automatic in the map view. The reverse is also true. If I select something on the bottom list view, the map will scroll automatically, but no the bottom view. Does it make sense?